### PR TITLE
Dcolligan 70 cfg file

### PR DESCRIPTION
**Pull request by dcolligan**
This works, but I'm still not sure what preference we want to override what; see discussion in #70

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Let's get #80 merged first before we discuss this.

I think it would be useful to give a little more information in the PR title - I have trouble remembering what issue is what! Also, I'm not sure putting the user name into the title adds much information.

**Comment by Naburimannu** [comment]
Agreed with Jerome's comments.

As I read the flask doc, the implicit expectation seems to be that paths are absolute, which would allow you to not bury your config file inside ga4gh/. I'd prefer not committing example.cfg at all.

I suspect that using DEBUG=True in your example *should* give a visible change in server behavior which would be easy to manually verify.

**Comment by pashields** [comment]
+1

**Comment by dcolligan** [comment]
@jeromekelleher Yes, I agree that we should wait on #80 before moving forward with this.  This pull request was more of a POC for the time being.  Also, good suggestion on more informative PR request titles.  Github seems to parse them automatically from the branch name if not provided.  I put the user name in as a matter of habit; at a previous job we had trouble identifying which branch belonged to whom but with Github's interface that doesn't really seem to be an issue.  I'll adopt a more useful convention.

**Comment by dcolligan** [comment]
Ok, so now that #80 has been merged, can we revisit this PR?

File: ga4gh/frontend.py [hunk 1]
```diff
@@ -15,11 +15,13 @@
 app = api.FlaskAPI(__name__)
 
 
-def configure(config):
+def configure(config, config_file):
```
Comment (jeromekelleher) [inline ga4gh/frontend.py]: It'll be easier to give this some default values. What about `configure(config="DefaultConfig", configFile=None)`?

File: tests/test_views.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 class TestFrontend(unittest.TestCase):
 
     def setUp(self):
-        frontend.configure("TestConfig")
+        frontend.configure("TestConfig", None)
```
Comment (jeromekelleher) [inline tests/test_views.py]: This change isn't needed with the default value for `configFile`.

**Comment by jeromekelleher** [comment]
Looks good to me; it's a useful tool setting up configurations for development. +1, modulo the comments above.

**Comment by dcolligan** [comment]
@jeromekelleher pushed commit with your suggestions

**Comment by jeromekelleher** [comment]
OK, two +1s so I'm merging. Thanks @dcolligan.

**Pull request merged**